relayer/src/index.ts

```diff
@@ -23,6 +29,22 @@ const createApi = async (url: string, types?: RegistryTypes) => {
   return api;
 };
 
+// TODO: consider moving to utils
+const sleep = (ms: number): Promise<void> =>
+  new Promise((resolve) => setTimeout(resolve, ms));
+
+const createFeeds = async (target: Target): Promise<number[]> => {
+  const feedIds: number[] = [];
+
+  for await (const _ of config.sourceChainUrls) {
+    await target.createFeed(feedIds);
```

Comment:
This is a strange API, what about returning `feedId` and pushing into `feedIds`? I think it should be possible to map `config.sourceChainUlrs` and then just `Promise.all()` at the end:

```typescript
return Promise.all(
    config.sourceChainUrls.map(() => target.createFeed()),
);
```

Reply:
yes, I have already managed to find a way to return a promise from subscription - it is implemented in another branch, will be included in the next PR